recipe/meta.yaml

```diff
@@ -9,57 +9,115 @@ source:
   - url: https://bitbucket.org/osrf/{{ name }}/get/{{ name }}{{ version.split('.')[0] }}_{{ version }}.tar.gz
     sha256: f1c873dfa17c1653332321ef4f10430fb22825de0688017455a2983f176cd69e
     patches:
-      - use-external-libs-config.patch
-      - qwt-fix.patch
-      - normalize-ogre-path.patch
+      - fix_build.patch  # [linux]
+      - use-external-libs-config.patch  # [win]
+      - qwt-fix.patch  # [win]
+      - normalize-ogre-path.patch  # [win]
 
 build:
-  number: 4
-  skip: true  # [not win]
+  number: 0
+  skip: true  # [not linux]
   run_exports:
     - {{ pin_subpackage('gazebo', max_pin='x.x') }}
 
 requirements:
   build:
-    - {{ compiler('cxx') }}              # [not win]
-    - {{ compiler('c') }}                # [not win]
-    - vs2017_win-64                      # [win64]
-    - vs2017_win-32                      # [win32]
-    - ninja
+    - {{ compiler('cxx') }}         # [not win]
+    - {{ compiler('c') }}           # [not win]
     - cmake
     - pkg-config
+    - ninja
+
+    # x11 stuff
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxcb') }}            # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxfixes') }}         # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('libxext') }}           # [linux]
+    - {{ cdt('libxau') }}            # [linux]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
   host:
-    - libsdformat 9.*
+    - xorg-libxext      # [linux]
+    - xorg-libxdmcp     # [linux]
+    - xorg-libxau       # [linux]
+    - libxcb            # [linux]
+    - xorg-libxfixes    # [linux]
+    - xorg-libxrender   # [linux]
+    - xorg-libx11       # [linux]
+    - xorg-libice       # [linux]
+    - xorg-libsm        # [linux]
+    - libglu            # [linux]
+    # deps
+    - libprotobuf
+    - libsdformat
     - libignition-transport8
-    - libignition-fuel-tools4
     - libignition-common3
+    - libignition-fuel-tools4
     - qt
-    - ogre
+    - ogre 1.12.5 h6508522_2  # [linux]
+    - ogre 1.12.5  # [win]
```

Comment:
@wolfv @traversaro Curious about have you run into any problems on Linux with OGRE 1.12? At least from my testing, it seems Gazebo is not really OGRE 1.12 ready and there are more work to do to enable that. I am thinking we may want to downgrade it to 1.10 for now...

Reply:
As far as I remember, I did not run Gazebo with Ogre 1.12.

Reply:
I opened a dedicated upstream issue to track this: https://bitbucket.org/osrf/gazebo/issues/2700/gazebo-11-does-not-run-correctly-with-ogre .